day_two_guide/topics/proc_restoring-etcd.adoc

```diff
@@ -11,253 +11,37 @@ Module included in the following assemblies:
 [id='restoring-etcd_{context}']
 = Restoring etcd
 
-The restore procedure for etcd configuration files replaces the appropriate
-files, then restarts the service or static pod.
+== Restoring the etcd configuration file
 
 If an etcd host has become corrupted and the `/etc/etcd/etcd.conf` file is lost,
-restore it using:
+restore it using the following procedure:
 
-----
-$ ssh master-0
-# cp /backup/yesterday/master-0-files/etcd.conf /etc/etcd/etcd.conf
-# restorecon -RvF /etc/etcd/etcd.conf
-ifeval::["{context}" != "downgrade"]
-endif::[]
-----
-
-In this example, the backup file is stored in the
-`/backup/yesterday/master-0-files/etcd.conf` path where it can be used as an
-external NFS share, S3 bucket, or other storage solution.
-
-[NOTE]
-====
-If you run etcd as a static pod, follow only the steps in that section. If you
-run etcd as a separate service on either master or standalone nodes, follow the
-steps to restore data as required.
-====
-
-ifeval::["{context}" != "downgrade"]
-== Restoring etcd data
```

Comment:
@geliu2016 / @hexfusion 

Removed this procedure. I _think_ it was only for v2 data.